pychromecast/discovery.py

```diff
@@ -90,7 +90,6 @@ def start_discovery(callback=None):
 
 def stop_discovery(browser):
     """Stop the chromecast discovery thread."""
-    browser.cancel()
```

Comment:
This was unnecessary since closing zeroconf also closes all listeners. Furthermore, this caused an exception if called in one of the callbacks, since it tried to join the discovery thread to itself.